tasks/connectors/veracode_asset_vulns/lib/veracode_av_client.rb

```diff
@@ -347,7 +386,7 @@ def issues(app_guid, app_name, tags, page_size, scan_types)
           if @assets.nil? || @assets.empty?
             print_good "No data for #{app_name}. Skipping Upload."
           else
-            kdi_upload(@output_dir, "veracode_#{fname}.json", @kenna_connector_id, @kenna_api_host, @kenna_api_key)
+            kdi_upload(@output_dir, "veracode_#{fname}.json", @kenna_connector_id, @kenna_api_host, @kenna_api_key, false, 3, 2)
```

Comment:
Please give the constants, false, 3, and 2, names so that the reader understands what is being passed into `kdi_upload()`.

Reply:
@dbrothersKenna  can you address this and also removed the old commented lines of code on your next check in.